c2c-extension/src/main/resources/c2c-schema.json

```
@@ -170,6 +193,7 @@
                     }
                   },
                   "config_name": {
+                    "description": "Name of the config config map",
```

```suggestion
                    "description": "Name of the config config map",
```
Is it correct to repeat the word `config` here?